README.md

```diff
@@ -36,6 +36,13 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 $ cp .env.template .env  # (first time only)
 ```
 
+Populate the .env file with the relevant values for:
+- TRELLO_API_KEY
+- TRELLO_API_TOKEN
+- BOARD_ID
+- LIST_ID
```

Comment:
Particularly given the non-standard label approach, it would be good to add a quick note that three labels with the names "Not Started", "In Progress" and "Complete"  need to be set up as that took me a few minutes to track down and get right

Reply:
this is very sensible.
since it's a slightly gross setup that I wouldn't want to keep once I've got a database it feels weird to write something so arbitrary in a readme!